app/uk/gov/hmrc/customs/emailfrontend/viewmodels/EmailVerifiedOrChangedViewModel.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.viewmodels
+
+import uk.gov.hmrc.customs.emailfrontend.config.AppConfig
+
+case class EmailVerifiedOrChangedViewModel(
+                                            email: String,
+                                            referrerUrl: Option[String],
+                                            isVerifyJourney: Boolean,
+                                            appConfig: AppConfig
+                                          ) {
+
+  def titleKey: String =
+    if (isVerifyJourney)  { "customs.emailfrontend.email-verified.title-and-heading" }
+    else { "customs.emailfrontend.email-changed.title-and-heading" }
+
+  def panelKey: String =
```

Comment:
I believe PanelKey is not used anywhere. also, could you find the message key. pls check if its used.